python/perf-kernels/06-fused-attention-fwd-transV.py

```diff
@@ -183,45 +195,49 @@ def forward(ctx, q, k, v, sm_scale):
 
 attention = _attention.apply
 
+name_to_torch_types = {
+    'fp16': torch.float16,
+    'fp8': float8
```

Comment:
what does the `float8` mean?

Reply:
It's defined as one of the fp8 format at the top of this file